subscription-manager.spec

```diff
@@ -296,6 +296,7 @@ rm -rf %{buildroot}
 # yum plugin config
 %config(noreplace) %attr(644,root,root) %{_sysconfdir}/yum/pluginconf.d/subscription-manager.conf
 %config(noreplace) %attr(644,root,root) %{_sysconfdir}/yum/pluginconf.d/product-id.conf
+%config(noreplace) %attr(644,root,root) %{_sysconfdir}/yum/pluginconf.d/search-disabled-repos.conf
```

Comment:
With depending on yum API_VERSION Requires is probably not necessary. I'm not sure what you mean by the second question.